Makefile

```diff
@@ -40,8 +40,8 @@ manifests:
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go rbac --name cloud-credential-operator
 	# kustomize and move to manifests dir for release image:
-	kustomize build config > manifests/0000_30_cloud-credential-operator_01_deployment.yaml
-	cp config/crds/cloudcredential_v1_credentialsrequest.yaml manifests/0000_30_cloud-credential-operator_00_v1_crd.yaml
+	kustomize build config > manifests/01_deployment.yaml
+	cp config/crds/cloudcredential_v1_credentialsrequest.yaml manifests/00_v1_crd.yaml
```

Comment:
The manifests are used for CVO release image ordering I thought, and the filenames were specific to the ordering we needed. Are we sure this is safe to change now? Maybe something has changed I'm not aware of.